Application/mobile/app_code/lib/pages/contact/conversation_state.dart

```diff
@@ -83,7 +83,6 @@ class ConversationPageState extends State<ConversationPage> {
         curveColor: context.select((ThemeProvider themeProvider) =>
             themeProvider.currentTheme.secondaryHeaderColor),
         showBackButton: false,
-        showLogo: true,
```

Comment:
You should add the title of the conversation to the app bar
It is available on ligne 95